pipelines/lib/util.groovy

```diff
@@ -1269,7 +1269,9 @@ def runDocumenteer(Map p) {
       dir(p.docTemplateDir) {
         bash '''
           source /opt/lsst/software/stack/loadLSST.bash
-          export PATH="${HOME}/.local/bin:${PATH}"
+          command -v dot || mamba  create -y --prefix=${HOME}/graphviz graphviz
+          export PATH="${HOME}/.local/bin:${HOME}/graphviz/bin:${PATH}"
```

Comment:
This is trying to combine two different conda environments in one.  It's not recommended and can lead to problems.  And then we're pip-installing on top of that.

Furthermore, you always change the PATH even if you don't install graphviz.

Reply:
Thanks. Will try the above.